### PR TITLE
Documentation fixes

### PR DESCRIPTION
**Pull request by dbrgn**
Some leftovers, fixes and improvements.

### CODE REVIEW

File: docs/src/config_directories.md [hunk 1]
```diff
@@ -11,7 +11,7 @@ exist, it will be created.
     [directories]
     cache_dir = "/home/myuser/.tealdeer-cache/"
 
-If no `cache_dir` is specified, tealdeer will fall back to a location that
```
Comment (niklasmohrin) [inline docs/src/config_directories.md:14]: Are you sure? So far I thought we have the lowercased word as the "brand name"

**Review by niklasmohrin — APPROVED** [review]

File: docs/src/installing.md [hunk 2]
```diff
@@ -40,17 +40,17 @@ Build and install the tool via cargo...
 
 ## Build From Source
 
-Debug build with logging enabled:
```
Comment (niklasmohrin) [inline docs/src/installing.md:47]: Hm, this reads as if you have to set it for building, not at run time; unless that really is the case, I think the previous note was better

File: docs/src/config_directories.md (hunk 1 above)
Comment (dbrgn) [reply docs/src/config_directories.md:14]: It's a bit inconsistent 🙈 In the docs, so far we always capitalized the name (except for this line above). In the README, we did not. I don't really mind either way, do you have a preference? I'd slightly tend towards capitalizing it as a proper name (looks less strange at the beginning of a sentence).

File: docs/src/installing.md (hunk 2 above)
Comment (dbrgn) [reply docs/src/installing.md:47]: Fair point. What about the fixupped version?

Comment (niklasmohrin) [reply docs/src/installing.md:47]: Looks good

File: docs/src/config_directories.md (hunk 1 above)
Comment (dbrgn) [reply docs/src/config_directories.md:14]: > I think I would go with lowercase in general, but capitalize at the beginning of a sentence.

Works for me.

**Pull request merged**